client/dashboard/summary-button/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+	Card,
+	CardBody,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Button,
+	Icon,
+} from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import CoreBadge from 'calypso/components/core/badge';
+import { SummaryButtonProps, SummaryButtonFieldProps } from './types';
+import './style.scss';
+
+const noop = () => {};
+
+function SummaryButton(
+	{
+		title,
+		to,
```

Comment:
Seeing that the ref type is `HTMLAnchorElement`, is the expectation that this component is always going to get a `to` prop? In other words, is this component always going to render an anchor link? 

In that case, calling it `button` may be confusing (cc @mirka who has been looking at standardising links in calypso)

Reply:
No. I've updated to `React.ForwardedRef< HTMLAnchorElement | HTMLButtonElement >`. It might be a bit confusing, but it's a wrapper of `Button` component which does the same thing - might render either a `button` or an `a` tag.